lib/widgets/mxc_image.dart

```diff
@@ -238,13 +238,20 @@ class _MxcImageState extends State<MxcImage>
 
   @override
   Widget build(BuildContext context) {
-    final imageWidget = widget.animated
+    Widget imageWidget = widget.animated
```

Comment:
Why don't you use `final`?

Reply:
<img width="333" alt="Screenshot 2023-11-24 at 15 59 35" src="https://github.com/linagora/twake-on-matrix/assets/43041967/d11a10af-7812-43d3-827c-e81654414175">